moped-editor/src/views/projects/projectView/ProjectActivityLog.js

```diff
@@ -128,6 +128,12 @@ const useLookupTables = (data) =>
     return lookupData;
   }, [data]);
 
+/** Fields which do not need to be rendered in the activity log */
+const CHANGE_FIELDS_TO_IGNORE = [
+  "updated_by_user_id",
+  "updated_at"
+];
+
```

Comment:
updated by user id and updated at are changed on every single update, they should not be treated as activities in the activity log